pt/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: news_post
+title: "Lançado o Ruby 2.4.0-preview2"
+author: "naruse"
+translator: "Filipe Rocha"
+date: 2016-09-08 18:00:00 +0900
+lang: pt
+---
+
+Temos o gosto de anunciar a versão 2.4.0-preview2 do Ruby.
```

Comment:
I think `prazer` instead of `gosto` is more suitable here.
(I'm reviewing as a Brazilian here, I saw you're from Portugal, please give me your thoughts @filiperocha)